uint8 uav_id
uint8 arm_command
uint8 game_stage
